api/src/main/java/org/openmrs/module/appointments/service/impl/AppointmentsServiceImpl.java

```diff
@@ -220,6 +228,13 @@ public List<Appointment> getAllAppointmentsReminder(String hours) {
 
     }
 
+    @Transactional
+    @Override
+    public List<Appointment> getAllAppointments(Date forDate, String status) {
```

Comment:
There should be unit tests for this that cover standard usage at the very least.

Reply:
Agreed.  Looks like DAO tests can be added to the existing AppointmentDaoImplIT class, and there's some sample test data you can use, and then the service tests would go in AppointmentServiceTest class.